tests/edit/test_main.py

```diff
@@ -45,13 +49,33 @@ def test_edit_page_updates_nav_bar(edit_page: Page) -> None:
 
 
 @pytest.mark.integration
-def test_edit_page_renders_heading(edit_page: Page) -> None:
+def test_edit_page_renders_heading(
+    edit_page: Page, extracted_activity: ExtractedActivity
+) -> None:
     page = edit_page
     heading = page.get_by_test_id("edit-heading")
-    page.screenshot(path="tests_edit_test_main-test_edit_page_renders_heading.png")
     expect(heading).to_be_visible()
+    page.screenshot(path="tests_edit_test_main-test_edit_page_renders_heading.png")
     assert re.match(r"Aktivität 1\s*DE", heading.inner_text())
```

Comment:
Maybe a bit more pywrightish.
```suggestion
    expect(heading).to_have_text(re.compile(r"Aktivität 1\s*DE"))
```